system/Test/Fabricator.php

```diff
@@ -545,6 +548,13 @@ public function create(int $count = null, bool $mock = false)
 				$ids[] = $id;
 				self::upCount($this->model->table);
 			}
+			else
+			{
+				throw new FrameworkException(lang('Fabricator.createFailed', [
+					$this->model->table,
+					implode(' ', $this->model->errors()),
+				]));
```

Comment:
The formatting issue is one of the reasons I don't like codesniffer as the CS here.

On another note, the current pattern for framework-specific exceptions is to use public static methods and then use the localization there. I am not aware of the reasons behind but I think it should be the same here. Or perhaps this is the start and let's use the `new` keyword from now on?

Reply:
@paulbalandan I created PHPStan rule for that at PR https://github.com/codeigniter4/CodeIgniter4/pull/3868

Reply:
I didn't do this initially because I didn't want to add more test code to the production section, but I think it still makes sense anyway and helps clean up that throw. Changed.